main/res/values/strings_not_translatable.xml

```diff
@@ -95,4 +95,8 @@
     <string name="ellipsis" translatable="false">…</string>
     <string name="triple_dash_vertical" translatable="false">┆</string>
 
+    <string name="equation_hint">a + b</string>
```

Comment:
Here you need to add `translatable="false"` to all the entries. Placing them in this file takes them out of our workflow fro translation, adding the attribute tells the tools to not complain about missing localisations.

Reply:
Okay, understood.